src/scales/scale.radialLinear.js

```diff
@@ -275,6 +276,33 @@ function drawPointLabels(scale) {
 	ctx.restore();
 }
 
+function drawBackgroundColors(scale) {
+
+	var opts = scale.options;
+	var outerDistance = scale.getDistanceFromCenterForValue(opts.ticks.reverse ? scale.min : scale.max);
+
+	for (var i = getValueCount(scale) - 1; i >= 0; i--) {
+		var outerPointPosition = scale.getPointPosition(i, outerDistance);
```

Comment:
The coding style we use in most of the project now is to put all variable declarations next to each other so that they get joined by `rollup-terser` when minimizing. So it'd be helpful to change to:

```
function drawBackgroundColors(scale) {
	var opts = scale.options;
	var outerDistance = scale.getDistanceFromCenterForValue(opts.ticks.reverse ? scale.min : scale.max);
	var ctx = scale.ctx;
	var i, outerPointPosition, previousOuterPointPosition, nextOuterPointPosition, previousOuterHalfway, nextOuterHalfway;

	for (i = getValueCount(scale) - 1; i >= 0; i--) {
		outerPointPosition = scale.getPointPosition(i, outerDistance);
		previousOuterPointPosition = scale.getPointPosition(i === 0 ? getValueCount(scale) - 1 : i - 1, outerDistance);
		nextOuterPointPosition = scale.getPointPosition(i === getValueCount(scale) - 1 ? 0 : i + 1, outerDistance);

		// Each color starts from the midpoint of the previous scale entry to the midpoint of the next
		previousOuterHalfway = {x: (previousOuterPointPosition.x + outerPointPosition.x) / 2, y: (previousOuterPointPosition.y + outerPointPosition.y) / 2};
		nextOuterHalfway = {x: (outerPointPosition.x + nextOuterPointPosition.x) / 2, y: (outerPointPosition.y + nextOuterPointPosition.y) / 2};

		ctx.beginPath();
		ctx.moveTo(scale.xCenter, scale.yCenter);
		ctx.lineTo(previousOuterHalfway.x, previousOuterHalfway.y);
		ctx.lineTo(outerPointPosition.x, outerPointPosition.y);
		ctx.lineTo(nextOuterHalfway.x, nextOuterHalfway.y);
		ctx.fillStyle = opts.backgroundColors[i];
		ctx.fill();
		ctx.closePath();
	}
}
```